pkg/secrethub/auth_aws.go

```diff
@@ -0,0 +1,62 @@
+package secrethub
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/secrethub/secrethub-go/internals/api"
+	"github.com/secrethub/secrethub-go/internals/auth"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sts"
+)
+
+type awsAuthService struct {
+	client client
+}
+
+func newAWSAuthService(client client) AuthMethodService {
+	return &awsAuthService{
+		client: client,
+	}
+}
+
+func (s awsAuthService) Authenticate() (auth.Authenticator, error) {
+	// Currently always use the eu-west-1 region.
+	region := endpoints.EuWest1RegionID
```

Comment:
Can we make this a constant and maybe place it at the top of the file, calling it `defaultAWSRegionForSTS` or something like that?